File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include<iostream>
using namespace std;

int main(){
    int index=0,max=0;
    for(int i=1;i<10;i++){
        int t;
        cin>>t;
        if(t>max){
            index=i;
            max=t;
        }
    }
    cout<<max<<"\n"<<index;
    return 0;
}

File: 백준/Bronze/2566. 최댓값/최댓값.py
import sys

grid = []

for _ in range(9):
    grid.append(list(map(int, sys.stdin.readline().strip().split())))

max_val = 0
row_val = 1
col_val = 1

for i, row in enumerate(grid):
    if max_val < max(row):
        max_val = max(row)
        row_val = i+1
        col_val = row.index(max(row)) + 1

print(max_val)
print(row_val, col_val)